src/Npgsql/NpgsqlBinaryExporter.cs

```diff
@@ -54,25 +54,30 @@ public TimeSpan Timeout
 
         #region Construction / Initialization
 
-        internal NpgsqlBinaryExporter(NpgsqlConnector connector, string copyToCommand)
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
Would rather bang-initialize _typeHandlerCache to null (the effect is the same, just less code noise)